charts/argo-workflows/scripts/update-image-tag.sh

```diff
@@ -37,6 +45,7 @@ elif [[ "${MANUAL_DEPLOY}" = true ]]; then
 # Relies on the assumption the IMAGE_TAG is a commit SHA
 elif [[ "${LATEST_GIT_SHA}" = "${IMAGE_TAG}" ]]; then
   auto_deploys="$(yq '.automatic_deploys_enabled' "${FILE}")"
+  add_tag
```

Comment:
We should only tag the image after we've successfully update the helm chart image tag value (on line 51).